app/views/collections/form.html.erb

```diff
@@ -1,3 +1,16 @@
+<% content_for :breadcrumbs do %>
+  <%= render Elements::BreadcrumbNavComponent.new(dashboard: true, admin: false) do |component| %>
+    <% if (current_page?(action: :new) || (controller.action_name == 'create')) %>
+      <% component.with_breadcrumb(text: I18n.t('collections.edit.no_title')) %>
```

Comment:
Since we're in a view here, I think you can invoke `t()` without any scoping, and that makes sure we can use Rails' I18n extensions.

```suggestion
      <% component.with_breadcrumb(text: t('collections.edit.no_title')) %>
```